core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/MultiCloudWorkspaceService.scala

```diff
@@ -426,6 +416,19 @@ class MultiCloudWorkspaceService(override val ctx: RawlsRequestContext,
           )
         )
       )
+
+      _ <- traceWithParent("createWDSInstance", parentContext) { _ =>
+        logger.info(
+          s"Creating WDS instance - workspace:'${workspaceRequest.toWorkspaceName.name}' - UUID:${workspaceId}"
```

Comment:
nit: this will only log the workspace name, not the namespace+name. `${workspaceRequest.toWorkspaceName}` should get you both.